driver_cpl/cime_config/buildnml

```diff
@@ -55,6 +55,57 @@ def _create_drv_namelists(case, infile, confdir, nmlgen, files):
     elif case.get_value('RUN_TYPE') == 'branch':
         config['run_type'] = 'branch'
 
+    cimeroot = case.get_value("CIMEROOT")
```

Comment:
I don't think that this code should be here, it's functionality has been moved in 5.2